### PR TITLE
Bug 2086675: No option to "add capacity" via the Installed Operators tab

### PR DESCRIPTION
**Pull request by SanjalKatiyar**
Screenshot from 2022-05-18 14-12-11
Screenshot from 2022-05-18 14-12-21
Screenshot from 2022-05-18 14-12-33

### CODE REVIEW

File: frontend/packages/ceph-storage-plugin/src/plugin.ts [hunk 1]
```diff
@@ -534,7 +534,7 @@ const plugin: Plugin<ConsumedExtensions> = [
       },
     },
     flags: {
-      disallowed: [OCS_INDEPENDENT_FLAG, MCG_STANDALONE, FEATURES.ADD_CAPACITY],
+      disallowed: [OCS_INDEPENDENT_FLAG, MCG_STANDALONE],
```
Comment (SanjalKatiyar) [inline frontend/packages/ceph-storage-plugin/src/plugin.ts:537]: 1) No need to backport to OCP 4.10, this flag is added to this extension in `release-4.10` also but CSV do not have this flag in ODF 4.10
2) If we want to disable from CSV, ptal: https://github.com/red-hat-storage/ocs-operator/pull/1681

Let me know which PR to close and which to use.

Comment (bipuladh) [reply frontend/packages/ceph-storage-plugin/src/plugin.ts:537]: Let's close this PR and remove it from the operator.

Comment (GowthamShanmugam) [reply frontend/packages/ceph-storage-plugin/src/plugin.ts:537]: let's remove it from both no point in keeping this dummy flag here.

Comment (SanjalKatiyar) [reply frontend/packages/ceph-storage-plugin/src/plugin.ts:537]: When we will have dynamic extension for this we can simply add that to `odf-console` and enable the flag from `ocs-operator` and we will not have to come to `console` and add this disallowed flag again.
So IMO we can close this PR ??

**Pull request closed without merging**